lib/response-parsers/index.js

```diff
@@ -6,9 +6,16 @@
 const { ensureXMLDoc } = require('oniyi-utils-xml');
 
 // internal modules
+const communityParser = require('./community-entry');
 const communityMembersParser = require('./community-members-entry');
 
 const responseParser = {
+  getCommunity: function communityEntryParserFunction(responseXML) {
```

Comment:
since this function doesn't need an execution context, can you use arrow function instead?
```
getCommunits: responseXML => communityParser(ensureXMLDoc(responseXML))
```